a8c-files.php

```diff
@@ -108,7 +108,7 @@ private function init_vip_filesystem() {
 
 	public function init_photon() {
 		// Limit to certain contexts for the initial testing and roll-out.
-		// This will be phased out and become the default eventually.
+		// This maybe phased out and become the default eventually.
```

Comment:
Thought on making it even more explicit?
```suggestion
		// This check may be phased out and Photon may become the default eventually.
```